src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,49 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  addMovie: (movie: Movie | null) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [erorr, setError] = useState(false);
```

Comment:
```suggestion
  const [isMovieLoadErorr, setSsMovieLoadErorr] = useState(false);
```
check boolean variables  naming